plugins/task-plugin/src/variable/project-path-variable-resolver.ts

```diff
@@ -0,0 +1,89 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable } from 'inversify';
+import Uri from 'vscode-uri';
+import * as che from '@eclipse-che/plugin';
+import * as theia from '@theia/plugin';
+import * as startPoint from '../task-plugin-backend';
+
+const VARIABLE_NAME = 'current.project.path';
+const SELECTED_CONTEXT_COMMAND = 'theia.plugin.workspace.selectedContext';
+const PROJECTS_ROOT_VARIABLE = 'CHE_PROJECTS_ROOT';
+const ERROR_MESSAGE_TEMPLATE = 'Can not resolve \'current.project.path\' variable.';
+/**
+ * Contributes the path for current project as a relative path to the first directory under the root workspace.
```

Comment:
Contributes the **variable for getting** path ... ?

Reply:
fixed